tests/integration/TestSuite.php

```diff
@@ -1,4 +1,5 @@
 <?php
```

Comment:
Oh... I thought you were replacing that so I didn't even pay attention to that. Never mind then.